style.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+$red: #ec5242;
+$grey: #d3d3d3;
+$black: #272a31;
```

Comment:
- I currently cannot see where these declarations are being used. I'll suggest you delete them because `You Ain't Gonna Need It(YAGNI)` 😁

Reply:
### **You’re right, the deletion was forgotten 😁😁**